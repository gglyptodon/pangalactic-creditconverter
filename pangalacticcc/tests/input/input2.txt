glob is I
prok is V
ish is X
tegj is L
glob glob Silver is 34 Credits
glob prok Gold is 57800 Credits
pish pish Iron is 3910 Credits
how much is pish tegj glob glob ?
how much is glob pish pish ?
how many Credits is glob prok Silver ?
how many Credits is glob prok Gold ?
how many Credits is glob prok Iron ?
how many Credits is pish prok Gold ?

how much wood could a woodchuck chuck if a woodchuck could chuck wood ?
