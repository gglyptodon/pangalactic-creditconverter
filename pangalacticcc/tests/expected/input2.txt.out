I don't understand this statement about units: pish pish Iron is 3910 Credits
pish could not be translated. tegj glob glob is 52
pish could not be translated. pish could not be translated. glob is 1
glob prok Silver is 68 Credits
glob prok Gold is 57800 Credits
This unit is unkown to me: Iron
Not everything could be translated to roman numerals: pish prok
I have no idea what you are talking about
